francis-fbalanon/homework_2_13/4-homework-99-bottles-of-beer/app.js

```diff
@@ -0,0 +1,42 @@
+/*
+  Write a script that prints the lyrics to "99 Bottles of Beer on the Wall" in the terminal. 
+  If you're unfamiliar with the song, you can [find the lyrics here](http://www.99-bottles-of-beer.net/lyrics.html).
+  
+  Be sure that all of your output includes proper spacing (no words or values running into each other).
+
+  Hint: You can make your output appear on multiple lines by using the string \n to insert a line break.
+  
+  For an extra challenge, improve this code to print "1 bottle" rather than "1 bottles." (Hint: read up on if/else
+  conditional statements at https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/if...else). 
+*/
+
+/*
+3 bottles of beer on the wall, 3 bottles of beer.
+Take one down and pass it around, 2 bottles of beer on the wall.
+
+2 bottles of beer on the wall, 2 bottles of beer.
+Take one down and pass it around, 1 bottle of beer on the wall.
+
+1 bottle of beer on the wall, 1 bottle of beer.
+Take one down and pass it around, no more bottles of beer on the wall.
+
+No more bottles of beer on the wall, no more bottles of beer. 
+Go to the store and buy some more, 99 bottles of beer on the wall.
+*/
+
+var a = " bottles of beer on the wall, ";
+var b = " bottles of beer. Take one down and pass it around, ";
+var c = " bottles of beer on the wall.";
+var d = "1 bottle of beer on the wall.";
+
+
+for (var i = 99; i > 0; i--) {
+  if (i > 2) {
+    console.log(i + a + i + b + (i - 1) + c);
+  } else if (i > 1) {
+    console.log("2 bottles of beer on the wall, 2 bottles of beer. Take one down and pass it around, 1 bottle of beer on the wall.");
+  } else {
+    console.log("1 bottle of beer on the wall, 1 bottle of beer. Take one down and pass it around, no more bottles of beer on the wall.")
```

Comment:
All that work to split up the song into variables and `d` never saw the light of day! Missed opportunity there. :P